ANSWERS.md

```diff
@@ -0,0 +1,6 @@
+1. What is the DOM?
+    The document object model or the entire document.
```

Comment:
Would have liked to see a more elaborate answers. It  is also important to note how the DOM is similar to a tree DS.